.github/workflows/pr.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        pyver: ['3.10', '3.12']
+        pyver: ['3.10', '3.13']
```

Comment:
We still want to keep 3.10?

Reply:
I think so, since pytket still supports 3.10.